client/rest-high-level/src/test/java/org/elasticsearch/client/RestHighLevelClientTests.java

```diff
@@ -711,8 +710,9 @@ public void testProvidedNamedXContents() {
         assertTrue(names.contains(ShrinkAction.NAME));
         assertTrue(names.contains(FreezeAction.NAME));
         assertTrue(names.contains(SetPriorityAction.NAME));
-        assertEquals(Integer.valueOf(1), categories.get(DataFrameAnalysis.class));
+        assertEquals(Integer.valueOf(2), categories.get(DataFrameAnalysis.class));
         assertTrue(names.contains(OutlierDetection.NAME.getPreferredName()));
+        assertTrue(names.contains(org.elasticsearch.client.ml.dataframe.Regression.NAME.getPreferredName()));
```

Comment:
Does it have to be fully-qualified?

Reply:
Yes, there is also the `Regression` object from the evaluation API.